content/hi/docs/contribute/localization.md

```diff
@@ -0,0 +1,435 @@
+मशीन जनित अनुवाद अपने आप में अपर्याप्त है। स्थानीयकरण के लिए व्यापक मानव की आवश्यकता है
+गुणवत्ता के न्यूनतम मानकों को पूरा करने के लिए समीक्षा करें।
+{{< /caution >}}
+
+व्याकरण और अर्थ में सटीकता सुनिश्चित करने के लिए, आपकी स्थानीयकरण टीम के सदस्यों को सावधानी से
+प्रकाशन से पहले सभी मशीन जनित अनुवादों की समीक्षा करें।
+
+### स्रोत फ़ाइलें
+
+स्थानीयकरण टीम द्वारा लक्षित एक विशिष्ट रिलीज से अंग्रेजी फाइलों पर आधारित होना चाहिए ।
+प्रत्येक स्थानीयकरण टीम तय कर सकती है कि किस रिलीज़ को लक्षित करना है जिसे target version_below कहा जाता है।
+
+अपने लक्षित संस्करण के लिए स्रोत फ़ाइलें खोजने के लिए:
+
+1. https://github.com/kubernetes/website पर कुबेरनेट्स वेबसाइट रिपॉजिटरी पर नेविगेट करें।
+2. निम्न तालिका से अपने लक्षित संस्करण के लिए एक शाखा का चयन करें:
+
+   लक्ष्य संस्करण | शाखा
+   -----|-----
+    नवीनतम संस्करण | [`main`](https://github.com/kubernetes/website/tree/main)
+   पिछला संस्करण | [`release-{{< skew prevMinorVersion >}}`](https://github.com/kubernetes/website/tree/release-{{< skew prevMinorVersion >}})
+   अगला संस्करण| [`dev-{{< skew nextMinorVersion >}}`](https://github.com/kubernetes/website/tree/dev-{{< skew nextMinorVersion >}})
+
+`मुख्य` शाखा वर्तमान रिलीज `{{< latest-version >}}` के लिए कंटेंट रखती है। रिलीज टीम
+अगली रिलीज से पहले एक `{{< release-branch >}}` शाखा बनाएगा: v{{< skew nextMinorVersion >}}.
+
+###  i18n में साइट स्ट्रिंग्स
+
+एक नई भाषा-विशिष्ट फ़ाइल में
+[`data/i18n/en/en.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml)
+कंटेंट स्थानीयकरण में शामिल होनी चाहिए. एक उदाहरण के रूप में जर्मन का उपयोग करना:`data/i18n/de/de.toml`.
+
+एक नई स्थानीयकरण निर्देशिका जोड़ें और `data/i18n/`में फ़ाइल करें। उदाहरण के लिए, जर्मन (`de`) के साथ:
+
+```bash
+mkdir -p data/i18n/de
+cp data/i18n/en/en.toml data/i18n/de/de.toml
+```
+
+अपने स्थानीयकरण के अनुरूप फ़ाइल के शीर्ष पर टिप्पणियों को संशोधित करें,
+फिर प्रत्येक स्ट्रिंग के मान का अनुवाद करें। उदाहरण के लिए, यह जर्मन भाषा है
+खोज प्रपत्र के लिए प्लेसहोल्डर (Placeholder) टेक्स्ट:
+```toml
+[ui_search_placeholder]
+other = "Suchen"
+```
+
+साइट स्ट्रिंग्स को स्थानीयकृत करने से आप साइट-व्यापी टेक्स्ट और सुविधाओं को कस्टमाइज़ कर सकते हैं: उदाहरण के लिए, कानूनी
+प्रत्येक पृष्ठ पर पाद लेख में कॉपीराइट पाठ है।
+
+### भाषा विशिष्ट शैली गाइड और शब्दावली
+
+कुछ भाषा टीमों की अपनी भाषा-विशिष्ट शैली मार्गदर्शिका और शब्दावली होती है।
+उदाहरण के लिए  [कोरियाई स्थानीयकरण गाइड](/ko/docs/contribute/localization_ko/) देखें ।
+
+### भाषा विशिष्ट ज़ूम मीटिंग
+
+यदि स्थानीयकरण परियोजना को एक अलग मीटिंग समय की आवश्यकता है, तो एक नई पुनरावर्ती ज़ूम मीटिंग और कैलेंडर आमंत्रण बनाने के लिए SIG Docs Co-Chair या Tech Lead से संपर्क करें। यह केवल तभी आवश्यक है जब
+टीम बनाए रखने के लिए काफी बड़ी है और एक अलग बैठक की आवश्यकता है।
+
+CNCF नीति के अनुसार, स्थानीयकरण टीमों को अपनी मीटिंग SIG डॉक्स YouTube प्लेलिस्ट पर अपलोड करनी होगी। एक SIG Docs Co-Chair या Tech Lead प्रक्रिया में तब तक मदद कर सकता है जब तक कि SIG Docs इसे स्वचालित नहीं कर देता।
+
+## Branching strategy
```

Comment:
Can we localize this heading? Maybe at least “strategy”?

Reply:
good catch @sftim  that needs to be localize